overlays/wwinit/rootfs/etc/localtime.ww

```diff
@@ -0,0 +1 @@
+{{ ImportLink "/etc/localtime" }}
```

Comment:
Can you move this to a new overlay? Something like `localtime` maybe, unless there's another one that makes sense.